.github/workflows/compatibility-test.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
     if: github.event.review.state == 'APPROVED'
     strategy:
       matrix:
-        go-version: ["1.15", "1.16", "1.17"]
+        go-version: ["1.18", "1.19", "1.20"]
```

Comment:
Probably should add versions 1.21, 1.22

Reply:
Extending [this comment](https://github.com/riandyrn/otelchi/pull/38#issuecomment-2067827322), for testing on Go 1.18 we only support two next Go version which until Go 1.20. We also avoid including an extra matrix strategy to avoid creating more jobs in Github Actions